### PR TITLE
test(js): index.test.js

### PR DESCRIPTION
**Pull request by debasishsahoo**
Add another test in test .js file

test#1

### CODE REVIEW

**Review by js55998 — COMMENTED** [review]
This is a New Code ,Need a New Code

**Comment by js55998** [comment]
> This is a New Code ,Need a New Code

Ok but I will add new Test code

**Comment by debasishsahoo** [comment]
Can you give me some brief about the new test case

**Comment by js55998** [comment]
base one test that failed into testing

File: test/index.test.js [hunk 1]
```diff
@@ -6,4 +6,8 @@ test('adds 1 + 2 to equal 3', () => {
 
 test('adds 5 + 9 to equal 14', () => {
     expect(add(5, 9)).toBe(14);
+});
+
+test('adds 5 + 10 to equal 15', () => {
+    expect(add(5, 10)).toBe(15);
 });
```
Comment (js55998) [inline test/index.test.js:13]: This code is Simmiler in privuas test case

Comment (debasishsahoo) [reply test/index.test.js:13]: But  What i need to do...

Comment (js55998) [reply test/index.test.js:13]: You need to put some thing to that is failed

Comment (debasishsahoo) [reply test/index.test.js:13]: ok i will try my best

**Review by js55998 — APPROVED** [review]

**Pull request merged**